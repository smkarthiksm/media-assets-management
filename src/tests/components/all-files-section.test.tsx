import '@testing-library/jest-dom';
import { renderWithProviders } from '../test-utility';
import AllFilesSectionComponent from '../../components/all-files-section/all-files-section';

describe('AllFilesSectionComponent', () => {
  test('should render', () => {
    const { container } = renderWithProviders(<AllFilesSectionComponent />);
    expect(container).toMatchSnapshot();
  });

  test('should be null when isLoaderVisible is true', () => {
    const { container } = renderWithProviders(<AllFilesSectionComponent />, {
      preloadedState: {
        loader: { isLoaderVisible: true },
      },
    });

    expect(container).toBeEmptyDOMElement();
  });
});
