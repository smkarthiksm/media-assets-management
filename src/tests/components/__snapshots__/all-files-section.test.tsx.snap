// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AllFilesModalsComponent should render all modals 1`] = `<div />`;

exports[`AllFilesSectionComponent should render 1`] = `<div />`;

exports[`AllFilesSectionComponent should render all modals 1`] = `<div />`;
